Integrations/ThinkstCanary/ThinkstCanary.yml

```diff
@@ -0,0 +1,209 @@
+commonfields:
+  id: Thinkst Canary
+  version: -1
+name: Thinkst Canary
+display: Thinkst Canary
+category: Deception
+description: By presenting itself as an apparently benign and legitimate service(s), the Canary draws the attention of
+  unwanted activity. When someone trips one of the Canary's triggers, an alert is sent to notify the responsible
+  parties so that action can be taken before valubale systems in your network are compromised.
+detaileddescription: |-
+  Generate your API Authentication Token from Canary Tools:
+  https://help.canary.tools/knowledge_base/topics/forget-logging-in-what-apis-does-canary-support
+configuration:
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Canary Server URL (e.g., https://***.canary.tools)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Authentication Token
+  name: auth_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Hours before first fetch to retrieve incidents
+  name: fetchDelta
+  defaultvalue: "24"
```

Comment:
this is not according to our convention, we should allow some thing like `24 hours` as we do in other integrations
have a look in recorded future as an example